src/expandableCalendar/index.js

```diff
@@ -212,6 +212,7 @@ class ExpandableCalendar extends Component {
     return days.length / 7;
   }
 
+  // TODO: this logic repeat itself in WeekCalendar - consider moving to a presenter
```

Comment:
We should do that, this way it will be clearer what is the aim of passing only the `props.markedDates` instead of `getMarkedDates` from ExpandableCalendar.

Reply:
I created a Card in the board for it. 
We need to address markedDates differently. 
Let's address it separately, I want to push this PR